azavea/cac-tripplanner
# Sml/style per mode stats

## Overview

Style new itinerary formats with per-mode travel stats.

Mocks: https://marvelapp.com/2jh0ge8/

- Style itinerary summary cards built in #855 
- Style itinerary detail footer built in #857 
- Use correct transit icon for transit toggle control
- Bonus: Update all transit-related icons with Material icons


### Demo

![desktop-list](https://user-images.githubusercontent.com/128699/28435824-9a05eab2-6d62-11e7-9719-71fd92bae28a.gif)

![desktop-detail](https://user-images.githubusercontent.com/128699/28435827-9db197b0-6d62-11e7-9f95-2a1c352df983.png)

![mobile](https://user-images.githubusercontent.com/128699/28435828-a02a845c-6d62-11e7-9799-e43a05eeaf0e.gif)


## Testing Instructions

 * Plan trips: walk, bike, walk + transit, bike + transit
 * Ensure itinerary summary cards render correctly in each case
 * Ensure itinerary detail footer renders correctly in mutli-modal cases
 * Mobile: Ensure carousel swipe renders and behaves correctly

Connects #855 #857 #849
